tools/tmuxtest.sh

```diff
@@ -27,28 +64,28 @@ die() {
     exit 1
 }
 
-function cleanup() {
-    [ -n "${TEMP}" ] && rm -rf ${TEMP}
-}
-
-function followlog() {
+followlog() {
     target=$1
     logfile=$2
     tmux split-window -v -t "${target}"
     tmux resize-pane -t "${target}" -y 10
     tmux send-keys -t "${target}" "touch ${logfile} && tail -f ${logfile}" C-m
 }
 
-TEMP=$(mktemp -d "/tmp/raiden.XXXXX")
-touch $TEMP/password
-GETHPORT=8101
+# the temp directory should be remove only after the tmux session is exited
+# cleanup() {
+#     [ -n "${TEMP}" ] && rm -rf ${TEMP}
```

Comment:
nope, to remember myself that that I tried that given approach and it did not work, but I want to find a proper fix